site/sfguides/src/deploying_custom_models_to_snowpark_model_registry/deploying_custom_models_to_snowpark_model_registry.md

```diff
@@ -0,0 +1,168 @@
+author: Mats Stellwall
+id: deploying_custom_models_to_snowpark_model_registry
+summary: Through this quickstart guide, you will learn how to deploy a custom model to the Snowpark Model Registry.
+<!--- Categories below should be hyphenated, i.e., Getting-Started. Do not leave blank. Visit site for available categories. -->
+categories: Getting-Started
+environments: web
+status: Draft 
```

Comment:
@sfc-gh-mstellwall Is it still in Draft mode? If not, change this to Published for it to be deployed.